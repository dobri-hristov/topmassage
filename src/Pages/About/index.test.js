import React from 'react'
import TestRenderer from 'react-test-renderer'
import AboutPage from './index.js'


describe('About page', () => {
    test('should render correctly', () => {
        const component = TestRenderer.create(<AboutPage />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})