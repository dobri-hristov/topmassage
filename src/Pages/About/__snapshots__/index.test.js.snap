// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page should render correctly 1`] = `
<div>
  <section
    className="us"
  >
    <h1
      className="title"
    >
      За нас
    </h1>
    <p>
      Човешкото тяло е съвършено. То носи знания и мъдрост и ни насочва как да се грижим за него. Болката и неразположенията са сигнали за почивка и презареждане. Ние сме мястото, където релаксацията е лечебна и спокойна предразполагаща атмосфера, професионален екип, висококачествени материали, етерични масла и аромати на професионални козметични марки.
    </p>
    <p>
      Масажите в център за масажи 
      <b>
        Top Massage
      </b>
       премахват болката и физическата тревожност. Те са краткият път към радостта и удоволствието без стрес и напрежение. Масажните терапии, които предлагаме, са съобразени с вашите персонални нужди и симптоми, за да можете да усетите истинската сила и облекчение!
    </p>
    <p>
      В студио за масаж 
      <b>
        Top Massage
      </b>
       тялото се чувства у дома си – релаксирано, тонизирано, обновено. Кожата си поема дъх, мускулите и ставите усещат благотворната сила на оздравителната енергия сред екзотични аромати, масла и есенции, които окуражават сетивата да усещат пълноценната наслада от живота. Ние познаваме нуждите на тялото и говорим на неговия език с професионализъм и внимание, затова знаем каква е точната грижа за всеки наш гост. 
    </p>
    <p>
      Ние сме с вас, когато усетите неразположение или болка в гърба, болка във врата, болка в раменете, болки в ставите, болки в ръцете или краката, за да премахнем оплакванията и да върнем усмивката на лицето ви. Предлагаме различни видове масажи, изпълнени от ловките, но деликатни ръце на добър масажист!
    </p>
    <p>
      Доверете се на силата на масажа и запазете час в студио за масаж 
      <b>
        Top Massage
      </b>
      , за да почувствате живителната енергия в собственото си успокоено и тонизирано тяло. Ние ще ви доставим не само удоволствие, но и ще ви подкрепим да постигнете оптимално здраве, баланс и хармония. 
    </p>
  </section>
  <section
    className="masseurs"
  >
    <div
      className="team"
    >
      <p>
        <b>
          Запознайте се с екипа
        </b>
      </p>
    </div>
    <article
      className="employee"
    >
      <img
        alt="Михаела Иванова"
        src="https://res.cloudinary.com/dfwwzfev2/image/upload/v1637842573/top-massage/masseurs/Mihaela_lvoe9a.jpg"
      />
      <h3
        className="employeeName"
      >
        Михаела Иванова
      </h3>
      <h5
        className="employeePosition"
      >
        Мениджър
      </h5>
      <p
        className="employeeInfo"
      >
        С магистърска степен специалност кинезитерапия, Михаела има дългогодишен опит както кинезитерапевт, консултант, 
 експерт и управител.Нейната сила и страст е в индивидуално диагностициране и прилагане на ефективни масажи
 за превенция и профилактика на Вашето здраве.
      </p>
    </article>
    <article
      className="employee"
    >
      <img
        alt="Ана-Мария Христова"
        src="https://res.cloudinary.com/dfwwzfev2/image/upload/v1637842573/top-massage/masseurs/AnaMaria_gkxl7h.jpg"
      />
      <h3
        className="employeeName"
      >
        Ана-Мария Христова
      </h3>
      <h5
        className="employeePosition"
      >
        Масажист терапевт
      </h5>
      <p
        className="employeeInfo"
      >
        Ана-Мария е завършила Националната спортна академия и има следдипломна квалификация 
 по „Класически, източноазиатски, антицелулитни и козметични масажи“.
 Оттогава тя работи в областта на здравето и красотата и се специализира в различни масажи.
      </p>
    </article>
    <article
      className="employee"
    >
      <img
        alt="Теодора Донева"
        src="https://res.cloudinary.com/dfwwzfev2/image/upload/v1637842573/top-massage/masseurs/Teodora_zoqmtm.jpg"
      />
      <h3
        className="employeeName"
      >
        Теодора Донева
      </h3>
      <h5
        className="employeePosition"
      >
        Kинезитерапевт/ Mасажист
      </h5>
      <p
        className="employeeInfo"
      >
        С опит в сферата на възстановителните процедури, Теодора е магистър по кинезитерапия от Националната спортна академия.
 Тя се радва да помага както на здрави хора, така и на хора с различни здравословни проблеми,
 за да им помогне да укрепят и възстановят здравето си чрез движение.
 Нейната сила е лечебният масаж.
      </p>
    </article>
    <article
      className="employee"
    >
      <img
        alt="Габриела Петков"
        src="https://res.cloudinary.com/dfwwzfev2/image/upload/v1637842573/top-massage/masseurs/Gabriela_i57cxa.jpg"
      />
      <h3
        className="employeeName"
      >
        Габриела Петков
      </h3>
      <h5
        className="employeePosition"
      >
        Масажист терапевт и консултант
      </h5>
      <p
        className="employeeInfo"
      >
        След като завършва Национална Спортна Академия с разширено обучение по различните видове масажи,
 Габриела осъзнава, че нейното признание е да помага на хората.Тя специализира в различни видове масажи - рефлексотерапия,
 класически,ароматерапия, дълбокотъканен, и много други.Индивидуалният подход към всеки един клиент е водещото в нейната практика.
      </p>
    </article>
  </section>
</div>
`;
